### DISCUSSION TITLE
Gamepad Input

### DISCUSSION THREAD

**Discussion by TonyHuff** [Brikwerk/nxbt#73] [General]
Excellent package. I am able to connect via a RaspberryPI 3B with no issues. I can send commands via the keyboard interface to my switch. 
I am trying to use my controller to record a macro. I don't see GamePad as an available input on the NXBT webpage.
Is there something that I am doing incorrectly?

**Comment by chubbybaby31** [comment]
Hello, I am trying to use the python API along with my Xbox controller to try and create a sort of adapter that runs when I boot up my Raspberry pi which will automatically connect my Xbox controller to the switch. It all works except for when I hold down a button. The press_buttons() function only allows you to press buttons and not hold them down. When I do hold a button down it registers as multiple clicks. I am not sure how to work around this. Any suggestions will be very helpful, thank you.
